Tests/SourceKitLSPTests/RangeFormattingTests.swift

```diff
@@ -0,0 +1,52 @@
+//===----------------------------------------------------------------------===//
+//
+// This source file is part of the Swift.org open source project
+//
+// Copyright (c) 2014 - 2023 Apple Inc. and the Swift project authors
+// Licensed under Apache License v2.0 with Runtime Library Exception
+//
+// See https://swift.org/LICENSE.txt for license information
+// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+//
+//===----------------------------------------------------------------------===//
+
+import LanguageServerProtocol
+import SKLogging
+import SKTestSupport
+import SourceKitLSP
+import XCTest
+
+final class RangeFormattingTests: XCTestCase {
+  func testSelectComment() async throws {
```

Comment:
One test for now, will copy more of swift-format's own range-formatting tests over here.

Reply:
I don’t think we need to re-test swift-format’s range formatting in SourceKit-LSP. I would add another test that formats multiple lines and maybe one where only part of the line is selected. That should cover all possible position conversion issues that might occur in SourceKit-LSP’s code base.

Meant to post this yesterday but apparently forgot to hit send.